packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -83,16 +80,16 @@ mixin SearchControllerMixin<T extends DataSearchStateMixin> {
 
   void _startDebounceTimer(
     String search, {
-    @required bool searchPreviousMatches,
+    required bool searchPreviousMatches,
   }) {
     searchInProgress = true;
 
     if (_searchDebounce?.isActive ?? false) {
-      _searchDebounce.cancel();
+      _searchDebounce!.cancel();
     }
 
     _searchDebounce = Timer(
-      search.isEmpty ? Duration.zero : debounceDelay,
+      search.isEmpty ? Duration.zero : debounceDelay!,
```

Comment:
add an assert at the beginning of this message that debounceDelay is not null.  This will add some null safe protection if `_startDebounceTimer` is ever called from anywhere else than it's current caller, which checks that debounceDelay is not null.

Reply:
done